app/src/main/java/edu/mills/heartoftaiwanese/activity/UIExtensions.kt

```diff
@@ -0,0 +1,25 @@
+package edu.mills.heartoftaiwanese.activity
+
+import android.app.Activity
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import androidx.fragment.app.Fragment
+
+/**
+ * Extension functions for UI
+ *
+ * https://stackoverflow.com/questions/41790357/close-hide-the-android-soft-keyboard-with-kotlin#45857155
```

Comment:
It's good that you list the source, although I'd put that in a `//` comment, since it's not part of the interface.